.travis.yml

```diff
@@ -17,15 +16,18 @@ env:
 
 matrix:
   fast_finish: true
-  allow_failures:
-    - os: osx
   include:
-    - python: 2.7
+    - sudo: required
+      python: 2.7
       env: ENV=lint
       before_script: TOX_ENV=lint
-    - python: 2.7
+    - sudo: required
+      python: 2.7
       env: ENV=docs
       before_script: TOX_ENV=docs-travis
+      before_install:
+        - sudo apt-get update -qq
```

Comment:
I'm not 100% sure if that's the root cause, but lint and docs jobs / tasks seems to have started getting stuck (https://travis-ci.org/apache/libcloud see build history, current build is stuck as well) after this change was merged (first I thought it might be an issue with Travis).

Will see if my change (17384cabeb99ae8fbf634d01f444f0f3b0b8c0d6) fixes that.